src/main/groovy/org/shipkit/internal/notes/header/HeaderRemover.java

```diff
@@ -0,0 +1,46 @@
+package org.shipkit.internal.notes.header;
+
+import java.io.File;
+import java.io.IOException;
+
+public class HeaderRemover {
```

Comment:
I are right. This was the suggestion of one person from the team. I will change it. The less 
I/O operation the better:)